pytest-embedded-arduino/pytest_embedded_arduino/serial.py

```diff
@@ -68,6 +69,9 @@ def __init__(self, attributes):
             default_kwargs['force'] = False
             default_kwargs['chip'] = self.app.target
 
+        if self.erase_flash:
+            default_kwargs['erase_all'] = True
```

Comment:
here should be `default_kwargs['erase_flash']`?

Reply:
This is the argument passed to `esptool`, it's called `erase_all`.  I tried with `erase_flash` to keep it consistent, but that's not recognized.

Reply:
ah, sorry. my mistake.